docs/4.0/docs/getting-started/use-sealos-registry.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 6
+---
+
+# Use sealos registry to store and distribute your cluster image
+
+sealos registry is a docker v2 registry which is used to store and distribute cluster image.
+
+The registry domain is `hub.sealos.cn`.
+
+## Login to sealos cloud and download your kubernetes config.yml
+
+You can register a free account in [sealos cloud](https://cloud.sealos.io/) first. 
+
+Then, download and save the config yaml, it will be used to log in the registry.
+
+## Login sealos registry
+
+```shell
+$ sealos login -k config.yml hub.sealos.cn
+```
+
+## Use image in sealos registry to accelerate download speeds
+
+Replace image domain to `hub.sealos.cn` when using sealos cli.
+
+Here is an example:
+```shell
+$ sealos run hub.sealos.cn/labring/kubernetes:v1.24.0 hub.sealos.cn/labring/calico:v3.22.1
+```
+
+## Push your cluster image to sealos registry
+
+First, you should create an organization in sealos cloud imagehub application.
+
+You can edit and apply this yaml at sealos cloud terminal application:
+
+```yaml
+apiVersion: imagehub.sealos.io/v1
+kind: Organization
+metadata:
+  name: your-organization-name
+spec:
+  name: your-organization-name # same as metadata.name
+  creator: your-uuid # find at sealos cloud page
+  manager: [ your-uuid ] # you can add other user as manager
```

Comment:
`Creator` and init `Manager` should not writed directly by user, but auto filled in controller to prevent safety?

Reply:
ok, add webhook orgMutator to do this, see this [issue](https://github.com/labring/sealos/issues/2308).

Reply:
this docs will not change until issue closs.

Reply:
done, add docs.